modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java

```diff
@@ -167,24 +167,12 @@ private void _validate(ObjectEntry objectEntry) {
 				String pathParameter = (String)values.get("pathParameter");
 
 				if (Objects.equals(method, Http.Method.POST)) {
-					Matcher curlyBraceMatcher = _curlyBracePattern.matcher(
-						pathInParameterString);
-
-					if (!Validator.isBlank(pathParameter) ||
-						curlyBraceMatcher.matches()) {
-
-						throw new ObjectEntryValuesException.InvalidObjectField(
-							null,
-							"Path parameters are not supported by POST API " +
-								"endpoints",
-							"path-parameters-are-not-supported-by-post-api-" +
-								"endpoints");
-					}
-
-					_validateRegularPath(objectEntry, pathString);
+					_validatePOSTAPIEndpoint(
```

Comment:
As I told you before, I think that we could validate each APIEndpoint internal's structure  individually  and then continue with the general flow of the common validations